src/api/queries/blockchainQueries.js

```diff
@@ -1,9 +1,16 @@
-ApiClient_.prototype.fetchActiveAddresses = function (slug, from, to) {
+ApiClient_.prototype.fetchActiveAddresses = function(slug, from, to) {
   var query = {
-    'query': '{\
-       dailyActiveAddresses(slug: "' + slug + '",\
-                            from: "' + toUTC_(from) + '",\
-                            to: "' + toUTC_(to) + '",\
+    query:
```

Comment:
Looks ugly for prettified code

Reply:
Hmm, you're right, maybe we should not prettify all files ?